charts/retool/templates/_helpers.tpl

```diff
@@ -44,21 +44,74 @@ Common labels
 */}}
 {{- define "retool.labels" -}}
 helm.sh/chart: {{ include "retool.chart" . }}
-{{ include "retool.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
what is AppVersion vs. .Chart.Version ?  (only diff from retool.telemetry.labels)

I see it's used by retool-temporal-services-helm only 👀

Reply:
pretty sure `.Chart.Version` is the chart version, and `.Chart.AppVersion` would be the retool version. except `.Chart.AppVersion` would come from `appVersion:` in Chart.yaml, which is optional and absent for our chart. so I think `.Chart.AppVersion` is effectively always null, and we don't use it to specify retool version anyway. docs: https://helm.sh/docs/topics/charts/

I didn't realize all this when I first wrote this here though, so I'm just going to take it out so it doesn't confuse us further.